index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inspiration&family=Nanum+Gothic:wght@400;700;800&display=swap"
+    rel="stylesheet">
+
+  <link rel="stylesheet" href="css/style.css">
+  <title>Albert Einstein Tribute Page</title>
+</head>
+
+<body id="home">
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+  <script src="js/index.js"></script>
+  <header id="header" class="navbar">
+    <div>
+      <h1 class="logo"><a href="#home">E = mc2</a></h1>
+    </div>
+    <ul>
+      <li><a href="#Life">Life</a></li>
+      <li><a href="#Career">Career</a></li>
+      <li><a href="#Awards">Awards</a></li>
+    </ul>
```

Comment:
- [ ] Please wrap your main navigation menu in a `nav` tag for semantic purposes. So it will be like this:
  ```html
  <nav>
    <ul>
      <li><a href="#Life">Life</a></li>
      <li><a href="#Career">Career</a></li>
      <li><a href="#Awards">Awards</a></li>
    </ul>
  </nav>
  ```
  Semantic tags are important because they make the HTML more comprehensible by better defining the different sections and layout of web pages. It makes web pages more informative and adaptable, allowing browsers and search engines to better interpret content.